oic.enumerationmap-content.json

```diff
@@ -1,5 +1,5 @@
 {
-  "supportedenumerations": ["aborted","active","airDry","armedAway","armedInstant","armedMaximum","armedNightStay","armedStay","boiling","brewing","cancelled","completed","down","dry","disabled","enabled","filterMaterial","grinding","idle","ink","inkBlack","inkCyan","inkMagenta","inkTricolour","inkYellow","keepwarm","notsupported","pause","pending","pendingHeld","preWash","processing","rinse","stopped","spin","testing","toner","tonerBlack","tonerCyan","tonerMagenta","tonerYellow","wash","wrinklePrevent"],
+  "supportedenumerations": ["aborted","active","airDry","armedAway","armedInstant","armedMaximum","armedNightStay","armedStay","aroma","artificalintelligence","auto","boiling","brewing","cancelled","circulating","cleaning","clothes","completed","cool","delicate","disabled","down","dry","dual","enabled","extended","fan","fast","filterMaterial","focused","grinding","heating","heavy","idle","ink","inkBlack","inkCyan","inkMagenta","inkTricolour","inkYellow","keepwarm","normal","notsupported","pause","pending","pendingHeld","permapress","preWash","processing","pure","quick","quiet","rinse","sectored","silent","sleep","smart","spot","steam","stopped","spin","testing","toner","tonerBlack","tonerCyan","tonerMagenta","tonerYellow","warm","wash","wet","wind","wrinklePrevent","zigzag"],
```

Comment:
can every item be on a single line, so that comparison for a subsequent version would be easier?

Reply:
I've pushed a change that includes all of the enums and displays them on a single screen (without the need to scroll).  This should be good enough as you can see them all at once.  One per line would actually also be hard to view.

Reply:
adding 1 single item next time will then show as a 1 line change per item.

Reply:
I get that it won't highlight the exact thing that changed, but as everything is now visible it'll be a fairly obvious visual difference and so relatively easy to identify the addition.